Best Time to Buy and Sell Stock II.py

```diff
@@ -0,0 +1,31 @@
+class Solution:
+    
+    """
+        
+        Name : Shahreen Shahjahan Psyche
+        Time : O(1)
```

Comment:
you are traversing all the elements in the array to find the max profit.

Reply:
Oh yes! I forgot to edit this one! Thanks for pointing out! It should definitely be O(N).